src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -47,13 +50,21 @@ public void setGuiSettings(GuiSettings guiSettings) {
         this.guiSettings = guiSettings;
     }
 
-    public Path getAddressBookFilePath() {
-        return addressBookFilePath;
+    public Path getFoodDiaryFilePath() {
+        return foodDiaryFilePath;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setFoodDiaryFilePath(Path foodDiaryFilePath) {
+        requireNonNull(foodDiaryFilePath);
+        this.foodDiaryFilePath = foodDiaryFilePath;
     }
 
-    public void setAddressBookFilePath(Path addressBookFilePath) {
-        requireNonNull(addressBookFilePath);
-        this.addressBookFilePath = addressBookFilePath;
+    public void setName(String name) {
```

Comment:
Just a gentle reminder that the application should be for a single user

Reply:
This is just to allow the single user to change his name as he likes.